recipe/meta.yaml

```diff
@@ -18,12 +18,16 @@ requirements:
     - python >=3.5
```

Comment:
This one too.
```suggestion
    - python >=3.7
```

Reply:
Fair enough, although I'm not sure what gets run under this python